assets/css/styleguide.scss

```diff
@@ -43,9 +43,31 @@ a#skipnav {
   }
 }
 
+/* Fixes anchor position when header is fixed */
+:target:before {
+  content: "";
+  display: block;
+  height: 75px;
+  margin-top: -75px;
```

Comment:
This weird css is here to make sure anchor tags line up with the bottom of the header, and not with the top of the browser since that means the h2 gets covered when header is fixed. Another approach is to use negative margin and an offsetting positive padding on the element itself.